Cod3rsGrowth.forms/FormEditarCarro.cs

```diff
@@ -70,21 +62,21 @@ private void AoClicarNoBotaoSalvarEdicaoCarro(object sender, EventArgs e)
 
                 _servicoCarro.Editar(carroEditado);
                 MessageBox.Show($"Carro com ID {_idDaEdicao} editado com sucesso!", "Editando carro");
-                this.Close();
+                Close();
```

Comment:
Você esta usando o Close() com e sem o this. Eu não lembro se precisa, usar o this. em nesse metodo, porém tem lugares no seu codigo que ele esta com this. e tem lugar que não, verifica se isso ta certo.

Reply:
Está funcionando certinho, no dia que fiz essa mudança, fiz porque você tinha falado para o Bruno que não precisava do this ai, que ele já reconhecia o forms que era para fechar.

Reply:
da um ctrl + shif + f e procura todas as ocorrencias do this.Close() e ver se tem algum que não foi removido o this.